src/client/app/shared/string-utils.service.ts

```diff
@@ -38,7 +38,7 @@ export class StringUtils {
 			return null;
 		}
 
-		return s.replace(/\w\S*/g, (txt) => {
+		return s.replace(/\b\w\S*/g, (txt) => {
 			return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
```

Comment:
substr(1) on a single character string returns an empty string so that case works fine.